getting-started/gce/startup-script.sh

```diff
@@ -0,0 +1,66 @@
+#! /bin/sh
```

Comment:
It looks like this script would fail if it's run twice. Can you confirm?

If startup script is touching the filesystem, e.g., checking out code or downloading something, they should first check existence of those files/resources and decide whether to overwrite or leave untouched.

Reply:
Updated. No issues no (the `gsutil cp` will overwrite the file).